ci/pipelines/deploy-to-production.yml

```diff
@@ -227,6 +235,13 @@ resources:
       repository: govukpay/adminusers
       variant: release
       <<: *aws_prod_config
+  - name: adminusers-ecr-registry-test
+    type: registry-image
+    icon: docker
+    source:
+      repository: govukpay/adminusers
+      variant: release
```

Comment:
Do we want `variant: release` here?

Reply:
I guess not. Deleted. [Re-triggered job](https://pay-cd.deploy.payments.service.gov.uk/teams/pay-deploy/pipelines/deploy-to-production/jobs/retag-adminusers-image-for-test-perf/builds/4) which still works.